CODEOWNERS

```diff
@@ -81,23 +81,26 @@ packages/video_player/video_player_android/**            @camsim99
 # Owned by ecosystem team for now during the wrapper evaluation.
 packages/webview_flutter/webview_flutter_android/**      @bparrishMines
 
-# - iOS
+# - Darwin
 packages/camera/camera_avfoundation/**                   @hellohuanlin
 packages/file_selector/file_selector_ios/**              @jmagman
-packages/google_maps_flutter/google_maps_flutter_ios/**  @hellohuanlin
-packages/google_sign_in/google_sign_in_ios/**            @vashworth
-packages/image_picker/image_picker_ios/**                @vashworth
+packages/file_selector/file_selector_macos/**            @jmagman
+packages/google_maps_flutter/google_maps_flutter_ios/**  @cbracken
+packages/google_sign_in/google_sign_in_ios/**            @vashworth @loic-sharma
+packages/image_picker/image_picker_ios/**                @vashworth @loic-sharma
+packages/image_picker/image_picker_macos/**              @jmagman
 packages/in_app_purchase/in_app_purchase_storekit/**     @louisehsu
 packages/ios_platform_images/**                          @jmagman
 packages/local_auth/local_auth_darwin/**                 @louisehsu
 packages/path_provider/path_provider_foundation/**       @jmagman
-packages/pigeon/**/ios/**/*                              @hellohuanlin
-packages/pointer_interceptor/pointer_interceptor_ios/**  @ditman
```

Comment:
@LouiseHsu suggested she take this one, are you cool with that @ditman?

Reply:
200% cool with it, thanks @LouiseHsu :)